freetextresponse/private/view.js

```diff
@@ -3,6 +3,7 @@ function FreeTextResponseView(runtime, element) {
 
     var $ = window.jQuery;
     var $element = $(element);
+    var $seq_content = $('#seq_content');
```

Comment:
We should give this a more generalized name.
One of the benefits to having a variable here is that we could update the selector if we want to store this elsewhere in the future.
In this case, if we updated the selector, the variable name would no longer make sense,
so we'd technically want to update it too.

Maybe `container` or something?
That's not a great name; any better ideas?